app/src/main/java/com/fightpandemics/home/HomeActivity.kt

```diff
@@ -56,5 +70,24 @@ class HomeActivity : BaseActivity(), HomeContract.View {
 
     private fun fabAction() {
         // TODO
+        if(isFabOpen){
+            fab.startAnimation(rotate_backward)
+            fabCreateAsOrg.startAnimation(fab_close)
+            fabCreateAsIndiv.startAnimation(fab_close)
+            fabCreateAsOrg.isClickable = false
+            fabCreateAsIndiv.isClickable = false
+            fabCreateAsIndiv.visibility = View.INVISIBLE
+            fabCreateAsOrg.visibility = View.INVISIBLE
+            isFabOpen = false
+        } else {
+            fab.startAnimation(rotate_forward)
+            fabCreateAsOrg.startAnimation(fab_open)
+            fabCreateAsIndiv.startAnimation(fab_open)
+            fabCreateAsOrg.isClickable = true
+            fabCreateAsIndiv.isClickable = true
+            fabCreateAsIndiv.visibility = View.VISIBLE
+            fabCreateAsOrg.visibility = View.VISIBLE
+            isFabOpen = true
+        }
```

Comment:
To follow clean code practice, I'd suggest creating one function for each fab state. e.g., `showOptions()` to contain the `if` content, and `hideOptions()` to contain the `else` content.